arcsi/model/user.py

```diff
@@ -14,7 +14,7 @@ class User(db.Model, UserMixin):
     butt_user = db.Column(db.String())
     butt_pw = db.Column(db.String(128))
     roles = db.relationship(
-        "Role", secondary=roles_users, backref=db.backref("users"), lazy="dynamic"
```

Comment:
Sorry, I should mark this PR as draft, or at least write a proper description about the modifications.
This change was suggested in new flask-security's [change log](https://github.com/Flask-Middleware/flask-security/blob/master/CHANGES.rst#version-310).
It was also a good test for the CI as you can see if I tried to update only for version 3.0.2 it went fine.
In case when I wanted to update for 3.2.0 it failed because it missed this change, with this change it passed again